app/controllers/mixins/vm_show_mixin.rb

```diff
@@ -70,6 +70,24 @@ def identify_vm
 
   private
 
+  def set_named_scope
+    @named_scope =
+      case miq_search_exp_fields
+      when ->(fields) { fields.include? "owned_by_current_ldap_group" } then :with_miq_group
```

Comment:
From https://github.com/ManageIQ/manageiq/pull/10290#issuecomment-239876947

> * set_named_scope has a case statement now.
>   - This is first to limit setting that scope to only when we need it for the MiqExpression, not all the time
>   - **This also sets up for a future PR where I will need to use a different scope for a different filter.**
>   - The proc + includes? is kinda smelly, but I couldn't think of a more terse way of representing that. Suggestions welcome.

I eventually need to support other scopes, hence the case statement, not a ternary.  That said, an `if` might work better at this point:

```ruby
@named_scope = 
  if miq_search_exp_fields.include? "owned_by_current_ldap_group"
    :with_miq_group
  elsif miq_search_exp_fields.include? "owned_by_current_user"
    :with_evm_owner # This scope to be added in a future PR
  end
```

V.S. what it will be:

```ruby
@named_scope =
  case miq_search_exp_fields
  when ->(f) { f.include? "owned_by_current_ldap_group" } then :with_miq_group
  when ->(f) { f.include? "owned_by_current_user" } then :with_evm_owner
  end
```